can_messages.asciipb

```diff
@@ -293,17 +293,32 @@ msg {
 
 msg {
   id: 28
-  source: DRIVER_CONTROLS
-  # target: TELEMETRY
-  msg_name: "steering angle"
+  source: DRIVER_CONTROLS_STEERING
```

Comment:
I assume you will be storing the previous state in your master, comparing with the newly received state, and then raising local events on the master as appropriate when the state has changed?

Reply:
yes the master board will constantly listen for state changes then raise events based on the new state.

Reply:
So the master will know the state of all the inputs? (ie. this is a bitset of all the possible analog and digital inputs on steering)

Reply:
The above seems to no longer be true. The message below indicates that events will be raised across CAN on the master, as opposed to the master being aware of the state of all the slaves' inputs at a particular instant.

Is this the intended behaviour now?

Reply:
@karlding  yeah this is the intended behaviour.  The events will be raised in center console (master).